.github/workflows/ci.yaml

```diff
@@ -103,6 +105,8 @@ jobs:
         run: |
           # Defaults
           integrations_glob=""
+          mariadb_groups="['mariadb']"
+          postgresql_groups="['postgresql']"
```

Comment:
Maybe the logic should be inverted now, and only run the tests if `recorder` is in the testable integrations?

Reply:
I think it's OK as it is.